ptf/tests/config/fdb_config.py

```diff
@@ -0,0 +1,56 @@
+import os
+import time
+import sys
+import inspect
+
+from config.constant import *
+from sai_thrift.sai_adapter import *
+from sai_base_utils import * # pylint: disable=wildcard-import; lgtm[py/polluting-import]
+
+class FdbT0Config(object):
+    """
+    Class use to make all the fdb configurations.
+    """
+    def create_fdb_mac_address_list(self, test):
+        test.mac_list = []
+        print("Generate mac address...")
+        test.mac0 = '00:00:00:00:00:11' 
```

Comment:
there should be two kinds of values, constant or variables.
If those values do not in a pattern or cannot be generated from a function, they should be set in Constant.py
If they can be generated from some fun by some pattern, then we can make them in the beginning, then refer then later.